deepomatic/api/client.py

```diff
@@ -28,16 +28,11 @@
 from deepomatic.api.resources.task import Task
 from deepomatic.api.resources.account import Account
 
-###############################################################################
-
-API_VERSION = 0.7
-
-###############################################################################
 
 class Client(object):
 
-    def __init__(self, app_id=None, api_key=None, verify_ssl=None, check_query_parameters=True, host=None, version=API_VERSION, user_agent_suffix='', pool_maxsize=20):
```

Comment:
From a documentation point of view it was good to have all these parameters explicitly listed here.
If we add proper inline doc we could replace them as you did with args & kwargs though.